.github/workflows/documentation.yml

```
@@ -55,13 +55,22 @@ jobs:
         env:
           EARTHDATA_USERNAME: ${{ secrets.EDL_USERNAME }}
           EARTHDATA_PASSWORD: ${{ secrets.EDL_PASSWORD }}
-
-      - name: Deploy
-        if: |
-          github.event_name == 'push'
-          && (github.ref == 'refs/heads/main' || github.ref == 'ref/heads/documentation')
-          && github.repository == 'nsidc/earthaccess'
-        uses: peaceiris/actions-gh-pages@v3
+      - name: Upload site artifact
+        uses: actions/upload-pages-artifact@v1
         with:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          publish_dir: ./site
+          path: ./site
+
+  deploy-documentation:
+    if: |
+      github.event_name == 'push'
+      && (github.ref == 'refs/heads/main' || github.ref == 'ref/heads/documentation')
+      && github.repository == 'nsidc/earthaccess'
+    runs-on: ubuntu-latest
+    needs: documentation
```

I think this needs to match the new name for the job that builds the docs

```suggestion
    needs: build-documentation
```

---

Sure does! Thanks :)